src/pages/index.js

```diff
@@ -53,38 +71,87 @@ const profilePopup = new PopupWithForm(
   config.popupProfileSelector,
   (userData) => {
     userInfo.setUserInfo(userData);
```

Comment:
` userInfo.setUserInfo(userData);`

You need to update `DOM` only in blocks `then` after a successful response from the server so that when there is an error it wouldn’t change


**Please, correct it everywhere**

Reply:
done